### DISCUSSION TITLE
Initial release of growatt2lorawan-v2

### DISCUSSION THREAD

**Discussion by matthias-bs** [jgromes/RadioLib#1197] [Show and tell]
[growatt2lorawan-v2](https://github.com/matthias-bs/growatt2lorawan-v2) &mdash; LoRaWAN Node for Growatt Photovoltaic Inverter Modbus Data Interface &mdash; has been released.
Major parts of it have been reused from [BresserWeatherSensorLW](https://github.com/matthias-bs/BresserWeatherSensorLW).

A new feature is the uplink message scheduling, which can be used if there are multiple uplink messages which shall be sent at different intervals. Depending on the current boot count, the message types due for uplink are selected and their corresponding `fPort` is chosen.

**Comment by StevenCellist** [comment]
Pretty cool! I was lazily scrolling around GitHub a couple days ago and already noticed your work on it. I'd have to check the brand & type of the solar panels on our roof to see if it's compatible :)
